shenyu-admin/src/test/java/org/apache/shenyu/admin/AbstractReflectGetterSetterTest.java

```diff
@@ -87,7 +86,7 @@
     protected abstract Class<?> getTargetClass();
 
     protected Set<String> getExcludeFields() {
-        return new HashSet<>(Arrays.asList("serialVersionUID"));
+        return new HashSet<>(Collections.singletonList("serialVersionUID"));
```

Comment:
Why modify this place, `java.util.Arrays.ArrayList` and `java.util.Collections.SingletonList` have the same parent class.

Reply:
it one one argument ,Arrays#asList(T...),
the result is the same